.yamato/project-tests.yml

```diff
@@ -41,7 +41,15 @@ test_{{ project.name }}_{{ editor }}_{{ platform.name }}:
     flavor: {{ platform.flavor}}
   commands:
     - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
-    - {% if platform.name == "centos" %}DISPLAY=:0 {% endif %}upm-ci project test -u {{ editor }} --project-path {{ project.path }} --type project-tests
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
```

Comment:
Why aren't you just doing something like `com.unity.netcode.gameobjects/Tests/Editor/Build/BuildTests.cs`?
A test case to create the build makes sure this works everywhere/for everyone, doesn't require yamato infra, and failures will be reported as a failed test instead of console/job spew :shrug:

Only thing you miss is the signing...

Reply:
The goal was not to just build but to also run the tests, the next PR will actually get the tests to run, I'm just trying to break what was a very large PR into a series of small changes.